spec/requests/api/v2/allocations_spec.rb

```diff
@@ -1,6 +1,17 @@
 require "rails_helper"
 
 RSpec.describe "/api/v2/allocations/<id>", type: :request do
+  before do
+    # Note: the concept of recruitment cycle here is ethereal.
+    # it is tied to a recruitment cycle that is associated to
+    # Settings.allocation_cycle_year
+    allocation_recruitment_cycle
```

Comment:
In the settings we call this `allocation cycle`. That makes more sense than `allocation recruitment cycle`, in my mind at least, should we use that or is the longer version correct?